cfgov/ask_cfpb/documents.py

```diff
@@ -0,0 +1,508 @@
+from django import forms
```

Comment:
we can move this file into `cfgov/ask_cfpb/models/search/documents.py`

Reply:
Done